clusterDeployer.py

```diff
@@ -187,19 +197,31 @@ def configure_bridge(h: host.Host, api_network: str) -> None:
 
     if "stp='off'" not in ret.out:
         logger.info("Destoying and recreating bridge")
-        logger.info("creating default-net.xml on localhost")
-        contents = """
+        logger.info(f"creating default-net.xml on {hostname}")
+        contents_without_ip_dhcp = """
 <network>
   <name>default</name>
   <forward mode='nat'/>
   <bridge name='virbr0' stp='off' delay='0'/>
+ """
+        contents_without_ip_dhcp_remaining = """
+</network>
+ """
+        content_ip_only = """
+  <ip address='192.168.123.250' netmask='255.255.0.0'/>
+  """
+        content_ip_dhcp = """
   <ip address='192.168.122.1' netmask='255.255.0.0'>
     <dhcp>
       <range start='192.168.122.129' end='192.168.122.254'/>
     </dhcp>
   </ip>
-</network>
 """
+        if hostname == "localhost":
+            contents = contents_without_ip_dhcp + content_ip_dhcp + contents_without_ip_dhcp_remaining
+        else:
+            contents = contents_without_ip_dhcp + content_ip_only + contents_without_ip_dhcp_remaining
+
```

Comment:
I need to be more clear. I was thinking something along the lines of having a function:
```
def network_xml(ip: str, dhcp_range: Optional[str]):
    if dhcp_range is None:
        dhcp_part = ""
    else:
        dhcp_part = f"<dhcp><range start='{dhcp_range[0]}' end='dhcp_range[1]}'/>"
    return f"""
<network>
  <name>default</name>
  <forward mode='nat'/>
  <bridge name='virbr0' stp='off' delay='0'/>
</network>
  <ip address='{ip}' netmask='255.255.0.0'>
    {dhcp_part}
  </ip>
</network>"
  """
```

Reply:
Small correction:

```
from typing import Tuple
```
and use
```
Optional[Tuple[int, int]]
```

Reply:
str*